Sources/Extensions/UIKit/UIViewController+Povio.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  UIViewController+Povio.swift
+//  PovioKit
+//
+//  Created by Toni Kocjan on 26/07/2020.
+//
+
+import UIKit
+
+public extension UIViewController {
+  class BarButton {
+    let content: Content
+    let action: Selector
+    let target: Any?
+    
+    required init(content: Content, action: Selector, target: Any? = nil) {
+      self.content = content
+      self.action = action
+      self.target = target
+    }
+  }
+  
+  enum Content {
+    case icon(UIImage)
+    case title(Title)
+    
+    static func icon(_ image: UIImage?) -> Content {
+      .icon(image ?? UIImage())
+    }
+  }
+  
+  enum Title {
+    case `default`(String)
+    case attributed(normal: NSAttributedString, disabled: NSAttributedString?)
+    
+    static func attributed(normal: NSAttributedString) -> Title {
+      .attributed(normal: normal, disabled: nil)
+    }
+  }
+}
+
+extension UIViewController {
+  @discardableResult
+  func setLeftBarButton(_ barButton: BarButton) -> UIBarButtonItem {
+    let button = createButton(using: barButton)
+    navigationItem.leftBarButtonItem = button
```

Comment:
This will only leverage on bar button per left/right item. Should we pre-pend items to `leftBarButtonItems` array instead? Or maybe even accept `barButtons: [BarButton]` as param 🤷‍♂️